force-app/main/default/classes/HOT_UpdateExistingRecords.cls

```diff
@@ -1,8 +1,12 @@
 public class HOT_UpdateExistingRecords implements Database.Batchable<sObject> {
     public Database.QueryLocator start(Database.BatchableContext bc) {
         Datetime now = Datetime.now();
+        // return Database.getQueryLocator(
+        //     'SELECT Id, HOT_Interpreters__c FROM WorkOrder WHERE (Status = \'Completed\' OR Status = \'Partially Complete\' OR Status = \'Dispatched\') AND HOT_Interpreters__c = null'
+        // );
+
         return Database.getQueryLocator(
-            'SELECT Id, HOT_Interpreters__c FROM WorkOrder WHERE (Status = \'Completed\' OR Status = \'Partially Complete\' OR Status = \'Dispatched\') AND HOT_Interpreters__c = null'
+            'SELECT Id, HOT_Interpreters__c FROM WorkOrder WHERE (Status = \'Completed\' AND ActualStartTime = null OR ActualEndTime = null'
```

Comment:
Pass på at status = Completed ALLTID må være sann

Reply:
Litt usikker på denne. Er en av disse 2 brukbare i dette tilfellet? Evt begge..?

1: 
WHERE (Status = \'Completed\' AND ActualStartTime = null AND ActualEndTime = null)

2:
WHERE Status = \'Completed\' AND (ActualStartTime = null OR ActualEndTime = null)